packages/commerce-sdk-react/package.json

```diff
@@ -38,6 +38,8 @@
   "types": "dist/index.d.ts",
   "private": true,
   "devDependencies": {
+    "@testing-library/jest-dom": "^5.16.4",
+    "@testing-library/react": "^12.1.5",
```

Comment:
FYI this version of `testing-library` can't be the latest 13, because it doesn't support our version of React in use. So I downgraded to 12.